tests/infra/base_tester.py

```diff
@@ -32,10 +32,12 @@ def __init__(
     ) -> None:
         self._comparison_config = comparison_config
 
-    @staticmethod
-    def _compile(executable: Callable) -> Callable:
+    def _compile(self, executable: Callable) -> Callable:
         """Sets up `executable` for just-in-time compile."""
-        return jax.jit(executable)
+        return jax.jit(executable, static_argnames=self._get_static_argnames())
```

Comment:
These `static args` are rather interesting, leaving docs [here](https://jax.readthedocs.io/en/latest/_autosummary/jax.jit.html) for reference.

We should not be providing `_get_static_argnames` in this base class for others to overload, mainly because overload mechanism is only used in model tests for now, ops and graphs don't need it.

What I suggest is to expand `_compile` method with additonal `static_argnames: Sequence[str]` parameter. Move the definition of `_get_static_argname` to `ModelTester`. Store them in some private variable like we did with others in `_init_model_hooks` and then pass them to `super()._compile`. 

I currently don't see a need for op and graph tests to use this feature, but if needed, we can easily expand `Workload` by one additional argument which will help us carry it wherever needed. Let's not do that for now, what I described above should suffice.

Reply:
Let's have a brief talk about this tomorrow

Reply:
Implemented the suggested changes